src/Dumplie/Application/Command/Customer/RemoveFromCartHandler.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Command\Customer;
+
+use Dumplie\Application\Transaction\Factory;
+use Dumplie\Domain\Customer\CartId;
+use Dumplie\Domain\Customer\Carts;
+use Dumplie\Domain\SharedKernel\Product\SKU;
+
+final class RemoveFromCartHandler
+{
+    /**
+     * @var Carts
+     */
+    private $carts;
+
+    /**
+     * @var Factory
+     */
+    private $factory;
+
+    /**
+     * @param Carts   $carts
+     * @param Factory $factory
+     */
+    public function __construct(Carts $carts, Factory $factory)
+    {
+        $this->carts = $carts;
+        $this->factory = $factory;
+    }
+
+    /**
+     * @param RemoveFromCart $command
+     *
+     * @throws \Exception
+     */
+    public function handle(RemoveFromCart $command)
+    {
+        $cart = $this->carts->getById(new CartId($command->cartId()));
+
+        $transaction = $this->factory->open();
+
+        try {
+            $cart->remove(new SKU($command->sku()));
```

Comment:
Some people (like me) prefer named constructors for VO, in this case it would be `SKU::fromString($command->sku());